content/en/docs/tasks/traffic-management/egress/egress-gateway/index.md

```diff
@@ -135,6 +135,13 @@ First create a `ServiceEntry` to allow direct traffic to an external service.
 1.  Create an egress `Gateway` for _edition.cnn.com_, port 80, and a destination rule for
     traffic directed to the egress gateway.
 
+    {{< warning >}}
+    When following these steps for additional hosts, do not create these resources again. Instead modify the resources
+    to work with multiple hosts. The `Gateway` resource accepts a list of hosts, or use `'*'` to match all. The `subset`
+    in the `DestinationRule` can be reused for additional hosts. Adding an additional `DestinationRule` targetting the
+    same Egress Gateway may result in undefined behavior.
+    {{< /warning >}}
```

Comment:
This isn't exactly true, and I think too much detail for here.
You can use more than one Gateway as long as you avoid [port name conflicts](https://preliminary.istio.io/latest/docs/ops/common-problems/network-issues/#port-conflict-when-configuring-multiple-tls-hosts-in-a-gateway).

Maybe a simple tip would be appropriate:
```suggestion
    {{< tip >}}
    To direct multiple hosts through an egress gateway, you can include a list of hosts, or use `'*'` to match all, in the `Gateway`.
    The  `subset` field in the `DestinationRule` can also be reused for additional hosts.
    {{< /tip >}}
```

Reply:
I've used this text as basis and changed wording a little bit for the DestinationRule. Our experience is that it all falls apart quickly when not reusing the subset, so made it a bit more 'demanding'